LEGO1/realtime/orientableroi.h

```diff
@@ -39,6 +39,7 @@ class OrientableROI : public ROI {
 	void FUN_100a58f0(const Matrix4& p_transform);
 	void FUN_100a5a30(const Vector3& p_world_velocity);
 
+	// FUNCTION: BETA10 0x1000fbf0
 	const Matrix4& GetLocal2World() const { return m_local2world; }
```

Comment:
Generally if a function has an annotation it should have a blank line after the function body, even if it's a one-liner (several more instances in the PR)